pkg/pool/pool.go

```diff
@@ -684,7 +684,7 @@ func (p *pool) getPDDeviation() int {
 	}
 
 	if deviationPrefix != 0 {
-		p.log.Info("calculating IP deviation for prefix pool", "p.warmPoolConfig", p.warmPoolConfig,
+		p.log.V(1).Info("calculating IP deviation for prefix pool", "p.warmPoolConfig", p.warmPoolConfig,
```

Comment:
nit: maybe change key to "warmPoolConfig" as 'p.' might not be relevant for logs.

Reply:
Sounds good, I've updated the name.